modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFRequest.macro

```diff
@@ -387,88 +387,104 @@ definition {
 	@summary = "Verify the fills done by the user on the Campaign Information"
 	macro viewMDFActivityDetails {
 		Click(
-			key_textButton = "Activity Name 1",
+			key_textButton = ${activityTitle},
```

Comment:
Hi @MGabriellaS, could you please replace `activityTitle` with '`campaignName`'? Just to maintain the variable pattern of the project